core/src/test/java/com/mapzen/android/search/MapzenSearchTest.java

```diff
@@ -18,16 +21,28 @@
 import retrofit2.Response;
 
 public class MapzenSearchTest {
+  private Context context;
+  private Pelias pelias;
+  private MapzenSearch search;
+
+  @Before public void setUp() throws Exception {
```

Comment:
nitpick, probably `setup` instead of camel-case `setUp`, unless there is some convention here.